packages/data-point/lib/entity-types/base-entity/resolve.js

```diff
@@ -7,16 +7,26 @@ const middleware = require('../../middleware')
 
 const utils = require('../../utils')
 
-function resolveErrorReducers (error, accumulator, resolveReducer) {
+/**
+ * @param {Object} manager
+ * @param {Function} resolveTransform
+ * @param {Object} accumulator
+ * @param {Error} error
+ * @returns {Promise<Accumulator>}
+ */
+function resolveErrorReducers (manager, resolveTransform, accumulator, error) {
```

Comment:
Changing this function signature was not strictly necessary, but this is my reasoning for the change:

`resolveErrorReducers` currently expects `resolveTransform` to have the manager parameter bound to it (for example by doing `_.partial(resolveTransform, manager)`. I feel like that's confusing, because when you see `resolveTransform` in different parts of the codebase, there might be confusion about whether or not that reference is bound or not